Do NOT apply this patch until abstract sockets are fully operational

--- origsrc/glib-2.36.4/gio/gunixsocketaddress.c	2013-08-07 09:34:32.000000000 -0500
+++ src/glib-2.36.4/gio/gunixsocketaddress.c	2013-10-21 13:09:30.530305400 -0500
@@ -41,7 +41,7 @@
  * However, some systems support abstract socket names which are not
  * visible in the filesystem and not affected by the filesystem
  * permissions, visibility, etc. Currently this is only supported
- * under Linux. If you attempt to use abstract sockets on other
+ * under Linux and Cygwin. If you attempt to use abstract sockets on other
  * systems, function calls may return %G_IO_ERROR_NOT_SUPPORTED
  * errors. You can use g_unix_socket_address_abstract_names_supported()
  * to see if abstract names are supported.
@@ -519,7 +519,7 @@ g_unix_socket_address_get_is_abstract (G
 gboolean
 g_unix_socket_address_abstract_names_supported (void)
 {
-#ifdef __linux__
+#if defined(__linux__) || defined(__CYGWIN__)
   return TRUE;
 #else
   return FALSE;
